docs/Installation/Installation.md

```diff
@@ -36,6 +36,7 @@ installation_on_clusters "Installation on clusters" page.
 * [Python](https://www.python.org/) 2.7, or 3.5 or later
 * [NumPy](http://www.numpy.org/) 1.10 or later
 * [SciPy](https://www.scipy.org)
+* [PyYAML](https://pyyaml.org)
```

Comment:
Agreed. However we will certainly need PyYAML to read input files.

Reply:
I already spend too much of my time fighting HPC systems so if this is the direction to go, I will no longer do that work because I know it'll take up too much of my time... I suggest you bring this up at the spectre call for broader discussion to see if anybody has any ideas on how to solve the problem :)

Reply:
I’m quite certain that wherever we have Python we also want PyYAML, to read input files. But we will only need Python for tests, docs and tools, at least for the foreseeable future. I think we plan to make Python entirely optional for production systems.

So my suggestion is: I move the PyYAML dependency to „Optional“, noting it is only needed for tests, docs and tools. And eventually we will move all Python dependencies there.

Reply:
How do you suggest we run tests on supercomputers if they have a bunch of dependencies that we cannot (easily) satisfy there? Not being able to run tests on supercomputers is a major problem since when we get access to a new system they're a good way to find any bugs on the system.

Reply:
I just checked and the python 3 env on Stampede2 does not have yaml.

Reply:
Discussed here https://github.com/sxs-collaboration/spectre-call-notes/issues/10